spec/loader_spec.rb

```diff
@@ -1,10 +1,57 @@
 require 'spec_helper'
 
 describe ActiveSetting::Loader do
+  let(:config_filename) { 'spec/settings.yml' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.